lib/metasploit/framework/parsed_options/console.rb

```diff
@@ -54,7 +53,11 @@ def option_parser
         end
 
         option_parser.on('-L', '--real-readline', 'Use the system Readline library instead of RbReadline') do
-          options.console.real_readline = true
+          message = "The RealReadline option has been marked as deprecated, and is currently a noop.\n"
+          message << "Metasploit Framework now uses Reline exclusively as the input handling library.\n"
+          message << "If you require this functionality, please raise an issue on GitHub:\n"
+          message << '  https://github.com/rapid7/metasploit-framework/issues/new?assignees=&labels=bug&projects=&template=bug_report.md'
```

Comment:
I'd probably pre-create this issue on our side and replace this link here, that way we have a single source of truth - instead of multiple disparate threads